config/apigroup_config.go

```diff
@@ -0,0 +1,714 @@
+		"azurerm_api_management_identity_provider_aad":                                   "apimanagement",
+		"azurerm_api_management_gateway":                                                 "apimanagement",
+		"azurerm_api_management_identity_provider_google":                                "apimanagement",
+		"azurerm_api_management_backend":                                                 "apimanagement",
+		"azurerm_api_management_api_diagnostic":                                          "apimanagement",
+		"azurerm_app_configuration_key":                                                  "appconfiguration",
+		"azurerm_app_configuration":                                                      "appconfiguration",
+		"azurerm_spring_cloud_custom_domain":                                             "appplatform",
+		"azurerm_spring_cloud_app_cosmosdb_association":                                  "appplatform",
+		"azurerm_spring_cloud_app_redis_association":                                     "appplatform",
+		"azurerm_spring_cloud_service":                                                   "appplatform",
+		"azurerm_spring_cloud_java_deployment":                                           "appplatform",
+		"azurerm_spring_cloud_app":                                                       "appplatform",
+		"azurerm_spring_cloud_active_deployment":                                         "appplatform",
+		"azurerm_spring_cloud_certificate":                                               "appplatform",
+		"azurerm_spring_cloud_app_mysql_association":                                     "appplatform",
+		"azurerm_private_dns_ptr_record":                                                 "arpa",
+		"azurerm_attestation_provider":                                                   "attestation",
+		"azurerm_role_definition":                                                        "authorization",
+		"azurerm_resource_group_policy_assignment":                                       "authorization",
+		"azurerm_policy_definition":                                                      "authorization",
+		"azurerm_management_group_policy_assignment":                                     "authorization",
+		"azurerm_resource_policy_assignment":                                             "authorization",
+		"azurerm_subscription_policy_assignment":                                         "authorization",
+		"azurerm_policy_set_definition":                                                  "authorization",
+		"azurerm_policy_assignment":                                                      "authorization",
+		"azurerm_role_assignment":                                                        "authorization",
+		"azurerm_management_lock":                                                        "authorization",
+		"azurerm_automation_module":                                                      "automation",
```

Comment:
Because a vast majority of these groupings (i.e., values of the map) are generated from Microsoft native provider names available in the Terraform registry as part of the import statements, it requires less manual work to include them. Also we are now using the native provider names as group names and not the prefix in the Terraform resource name as discussed in this [comment](https://github.com/crossplane-contrib/provider-jet-azure/pull/118#discussion_r778595791), which decreases the suitability of the default chosen by terrajet.

Reply:
Got it. I just tried it out and it validated your assumption, it was able to decrease it from 692 to only 503.